m_etcd/const.go

```diff
@@ -1,6 +1,9 @@
 package m_etcd
 
 const (
+	DefaultClaimTTL uint64 = 180 // 3 minutes in seconds
```

Comment:
What is the effect of changing these? How does it help/hurt that it's now `180` rather than `120` seconds?